packages/terra-horizontal-divider/src/HorizontalDivider.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './HorizontalDivider.scss';
+
+const cx = classNames.bind(styles);
+
+const HorizontalDivider = ({ ...customProps }) => {
+  const HorizontalDividerClassNames = cx([
+    'divider',
```

Comment:
Include the customProps.className in here

Reply:
Updated  https://github.com/cerner/terra-core/commit/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-25e37f5cb806bb82d51633a0117496a5R11